e2e-tests/cypress/e2e/journeys/insurance/page-not-found-when-logged-out.spec.js

```diff
@@ -0,0 +1,36 @@
+import { PAGES } from '../../../../content-strings';
+import { ROUTES } from '../../../../constants';
+
+const CONTENT_STRINGS = PAGES.PAGE_NOT_FOUND_PAGE;
+
+context('Insurance - page not found - logged out', () => {
```

Comment:
Can we rename this file and test description to be "signed out" instead of "logged out"